docs/application/native/guides/connectivity/nfc.md

```diff
@@ -98,6 +98,13 @@ The main features of the NFC API include:
 
   ![HCE routing](./media/nfc_hce_routing.png)
 
+  > **Note**
+  >
+  > Pay attention to the following:
```

Comment:
@jh8801jung 

Global comment: 

- Remove the instances of sentence **Pay attention to the following:** in whole document, as we are already giving a **Note**.
- Note alone serves the said purpose.